proto/pocket/application/tx.proto

```diff
@@ -2,16 +2,19 @@ syntax = "proto3";
 
 package pocket.application;
 
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
 option go_package = "pocket/x/application/types";
 
 // Msg defines the Msg service.
 service Msg {
-  rpc StakeApplication (MsgStakeApplication) returns (MsgStakeApplicationResponse);
+  rpc StakeApplication   (MsgStakeApplication  ) returns (MsgStakeApplicationResponse  );
   rpc UnstakeApplication (MsgUnstakeApplication) returns (MsgUnstakeApplicationResponse);
 }
-
 message MsgStakeApplication {
-  string address = 1;
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the application
```

Comment:
Should we be applying this option to the "sender" fields of all cosmos-sdk message types (i.e. `MsgCreateClaim`, `MsgSubmitProof`)?

Reply:
@bryanchriswhite I think anytime we use a bech32 encoded string this ought to be used.